theme.js

```diff
@@ -137,17 +112,6 @@ define(function(require, exports, module) {
             }
         }
 
-        plugin.freezePublicAPI({
-
-            /**
-             * Hides theme button.
-             */
-            hideButton: hideButton,
-
-            /**
-             * Shows theme button.
-             */
-            showButton: showButton,
-        });
```

Comment:
We're removing the theme button altogether, so we should remove this dependency from presentation!